src/encoded/static/components/item.js

```diff
@@ -6,39 +6,41 @@ var globals = require('./globals');
 var audit = require('./audit');
 var form = require('./form');
 var _ = require('underscore');
-var cx = require('react/lib/cx');
-var panel = require('../libs/bootstrap/panel');
+import { Panel } from 'react-bootstrap';
 var AuditIndicators = audit.AuditIndicators;
 var AuditDetail = audit.AuditDetail;
 var AuditMixin = audit.AuditMixin;
 var JSONSchemaForm = form.JSONSchemaForm;
 var Table = collection.Table;
-var {Panel, PanelBody, PanelHeading} = panel;
+var ErrorPage = require('./error');
 
 var Fallback = module.exports.Fallback = React.createClass({
     contextTypes: {
         location_href: React.PropTypes.string
     },
 
-    render: function() {
-        var url = require('url');
-        var context = this.props.context;
-        var title = typeof context.title == "string" ? context.title : url.parse(this.context.location_href).path;
-        return (
-            <div className="view-item">
-                <header className="row">
-                    <div className="col-sm-12">
-                        <h2>{title}</h2>
-                    </div>
-                </header>
-                {typeof context.description == "string" ? <p className="description">{context.description}</p> : null}
-                <section className="view-detail panel">
-                    <div className="container">
-                        <pre>{JSON.stringify(context, null, 4)}</pre>
-                    </div>
-                </section>
-            </div>
-        );
+    // render: function() {
```

Comment:
the nice thing about this error page vs the basic error page that was put in that just says page not found is that this error page provides a bit more details about what the error might actually be.  I would thus for for keeping the "old style" error page in so we get more descriptive info.